public/app/plugins/datasource/prometheus/dashboards/prometheus_2_stats.json

```diff
@@ -0,0 +1,1758 @@
+              "format": "time_series",
+              "hide": false,
+              "interval": "",
+              "intervalFactor": 2,
+              "legendFormat": "misc",
+              "refId": "F",
+              "step": 60
+            },
+            {
+              "expr": "sum(node_memory_MemFree{instance=~'[[node]].*', _ftio=\"\"})",
+              "format": "time_series",
+              "hide": false,
+              "interval": "",
+              "intervalFactor": 2,
+              "legendFormat": "free",
+              "metric": "",
+              "refId": "B",
+              "step": 60
+            },
+            {
+              "expr": "sum(node_memory_SwapTotal{instance=~'[[node]].*', _ftio=\"\"}) - sum(node_memory_SwapFree{instance=~'[[node]].*', _ftio=\"\"})",
+              "format": "time_series",
+              "interval": "",
+              "intervalFactor": 2,
+              "legendFormat": "swapped",
+              "refId": "G",
+              "step": 60
+            },
+            {
+              "expr": "sum(node_memory_MemTotal{instance=~'[[node]].*', _ftio=\"\"}) ",
+              "format": "time_series",
+              "interval": "",
+              "intervalFactor": 2,
+              "legendFormat": "total",
+              "metric": "",
+              "refId": "D",
+              "step": 60
+            }
+          ],
+          "thresholds": [],
+          "timeFrom": null,
+          "timeShift": null,
+          "title": "Host Memory",
+          "tooltip": {
+            "msResolution": false,
+            "shared": true,
+            "sort": 2,
+            "value_type": "individual"
+          },
+          "type": "graph",
+          "xaxis": {
+            "buckets": null,
+            "mode": "time",
+            "name": null,
+            "show": true,
+            "values": []
+          },
+          "yaxes": [
+            {
+              "format": "bytes",
+              "label": "GB",
+              "logBase": 1,
+              "max": null,
+              "min": "0",
+              "show": true
+            },
+            {
+              "format": "short",
+              "logBase": 1,
+              "max": null,
+              "min": null,
+              "show": true
+            }
+          ]
+        }
+      ],
+      "repeat": null,
+      "repeatIteration": null,
+      "repeatRowId": null,
+      "showTitle": false,
+      "title": "Dashboard Row",
+      "titleSize": "h6"
+    }
+  ],
+  "schemaVersion": 14,
+  "style": "dark",
+  "tags": [
+    "prometheus"
+  ],
+  "templating": {
+    "list": [
+      {
+        "allValue": null,
+        "current": {},
+        "datasource": "${DS_LOCAL-P8S}",
+        "hide": 0,
+        "includeAll": false,
+        "label": "Node",
+        "multi": false,
+        "name": "node",
+        "options": [],
+        "query": "label_values(prometheus_target_interval_length_seconds_count, instance)",
+        "refresh": 2,
+        "regex": "",
```

Comment:
node template maybe could use a regex like `"regex": "(.*):\\d+",`,
prometheus exporter path and node_exporter exporter path always with  diffrent port, will cause `instance=~'[[node]].*'` can not get right node address